iquip/apps/builder.py

```diff
@@ -400,52 +400,52 @@ def __init__(
         ):
         """Extended.
 
-        Args: See the docstring of Attributes for more detail.
+        Args: See the attributes section for more details.
             procdesc: Each key and its value are as follows.
```

Comment:
I noticed that the docstrings of all items are duplicated with `_ScanEntry.__init__()`.
So, you should remove the duplicated ones.

Also, please match the docstrings to the existing format exactly.